lib/Models/UserDrawing.ts

```diff
@@ -150,14 +152,6 @@ export default class UserDrawing extends MappableMixin(
     this.drawRectangle = defaultValue(options.drawRectangle, false);
 
     this.invisible = options.invisible;
-
-    // helper for dragging points around
-    this.dragHelper = new DragPoints(options.terria, (customDataSource) => {
-      if (typeof this.onPointMoved === "function") {
-        this.onPointMoved(customDataSource);
-      }
-      this.prepareToAddNewPoint();
-    });
   }
```

Comment:
Instead of creating the `dragHelper` in the constructor, we will create in only when the `UserDrawing` is activated. This will ensure that it'll capture the mouse only when it is being used.